### DISCUSSION TITLE
Meal planner - multiple users

### DISCUSSION THREAD

**Discussion by Selduro** [mealie-recipes/mealie#2500] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

There are multiple people in my household who don't always eat the same food for lunch. This is hard to display in the current meal planner.  Right now we are using the meal-tags for that. So "breakfast" is my lunch and "dinner" is my wife's lunch. We use the tag "lunch" if it's identical.

### Please provide a concise description of the feature that would resolve your issue.

I'd like the option to add multiple users in the meal planner and assign foods per day per user.

For example:
|                          | 14.8.  | 15.8.  | 16.8.  |
| ------------------------ | ------ | ------ | ------ |
| [If Meals are identical] |        | Food C |        |
| John                     | Food A |        | Food D |
| Jane                     | Food B |        | Food E |

### Please consider and list out some caveats or tradeoffs made in your design decision

I don't see any.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by bedge117** [comment]
I second this!

I have multiple users in the same group, and I would like the ability to have different meal planner schedules.

In addition to what Selduro is requesting, Per User would be great, or simply have the ability to create a second Meal Planner.

Meal Planner 1 (Custom Name) and Meal Planner 2 (Custom Name)

Something like that.
